tests/unit/certGen.unitTest.ts

```diff
@@ -1788,6 +1788,13 @@ describe("cert-gen", () => {
             const hgvFailWithMajorDefectDangerousRectified: any = JSON.parse(failWithPrsEvent.Records[4].body);
             const psvPrsNotAcceptableForBilingualCert: any = JSON.parse(prsEvent.Records[0].body);
 
+            const psvFailWithDefects: any = JSON.parse(event.Records[19].body)
+            // const psvFailWithMajorDefect: any = JSON.parse(...);
```

Comment:
Do we need these consts if they are commented out? can they be removed?

Reply:
Agree with Leo, if you don't want to use these examples, can you remove them please.

Also can you chuck a ; at the end of line 1791 please :)